Sources/Speechly/Audio/AudioRecorder.swift

```diff
@@ -152,7 +152,7 @@ public class AudioRecorder {
         }
 
         let audioSession = AVAudioSession.sharedInstance()
-        try audioSession.setCategory(.record, mode: .voiceChat)
+        try audioSession.setCategory(.playAndRecord, mode: .voiceChat)
```

Comment:
Why both play and record, shouldn't record be enough?

Reply:
I couldn't make it work initially until I changed this, it always failed at this row. Don't know why, but this seemed like a necessary change.

Reply:
Ok, let's go with this 👍